src/main/scala/org/scalabridge/sitegen/StaticSiteGenerator.scala

```diff
@@ -37,7 +55,7 @@ object StaticSiteGenerator {
     case H3(value)                => mkHtml(value, "h3")
     case Bold(value)              => mkHtml(value, "strong")
     case Italic(value)            => mkHtml(value, "em")
-    case Link(value)              => mkHtml(value, "a")
+    case Link(text, url)          => LinkHtml(text = text, url = url)
```

Comment:
I think the parameter names from the case match are probably sufficient here

```suggestion
    case Link(text, url)          => LinkHtml(text, url)
```

Reply:
@Lgoldfinch The thing is that here we have multiple parameters of the same type in a row. It's easy to accidentally substitute them in a wrong order upon refactoring. Named parameters is a standard tool in Scala to avoid such danger. But I don't insist.